README.md

```diff
@@ -130,6 +130,19 @@ Crate features:
   `RUST_LOG`.
 
 
+### Alternative Rust crates for property testing
+
+The [`proptest`](https://docs.rs/proptest) crate is inspired by the
+[Hypothesis](http://hypothesis.works) framework for Python.
+You can read a comparison between `proptest` and `quickcheck`
+[here](https://docs.rs/proptest/0.3.1/proptest/#differences-between-quickcheck-and-proptest)
```

Comment:
Maybe this should use `*` for the version so it always goes to the latest docs? I just released 0.3.2, so if not, it should be updated to point to that version.

Reply:
Neat! I didn't know `*` was a thing. Fixed.